lib/generators/alchemy/solidus/install/install_generator.rb

```diff
@@ -123,25 +123,25 @@ def inject_routes
               default: mountpoint
             )
         end
-        if File.read(routes_file_path).match SPREE_MOUNT_REGEXP
-          sentinel = SPREE_MOUNT_REGEXP
+        sentinel = if File.read(routes_file_path).match SPREE_MOUNT_REGEXP
+          SPREE_MOUNT_REGEXP
         else
-          sentinel = "Rails.application.routes.draw do\n"
+          "Rails.application.routes.draw do\n"
         end
-        inject_into_file routes_file_path, { after: sentinel } do
+        inject_into_file routes_file_path, {after: sentinel} do
           "\n  mount Alchemy::Engine, at: '/#{mountpoint.chomp("/")}'\n"
         end
       end
 
       def set_root_route
         routes_file_path = Rails.root.join("config", "routes.rb")
         if options[:auto_accept] ||
-             yes?("\nDo you want Alchemy to handle the root route '/'? (y/n)")
+            yes?("\nDo you want Alchemy to handle the root route '/'? (y/n)")
           sentinel = "Rails.application.routes.draw do\n"
-          inject_into_file routes_file_path, { after: sentinel } do
+          inject_into_file routes_file_path, {after: sentinel} do
             <<~ROOT_ROUTE
-              \  # Let AlchemyCMS handle the root route
-              \  root to: 'alchemy/pages#index'
+              # Let AlchemyCMS handle the root route
+              root to: 'alchemy/pages#index'
```

Comment:
This is the change that causes the specs to fail. Without the leading backslashes, the two lines have the wrong indentation, and the install generator does not realize the string is already there, leading to `root` being called twice.

Reply:
WOW! Good eyes

Reply:
Extracted the Rubocop changes into dedicated PR

Reply:
> WOW! Good eyes

Nah I didn't *see* that. I debugged it :)